contracts/multichain/MultichainEventUtils.sol

```diff
@@ -16,56 +16,56 @@ library MultichainEventUtils {
     function emitMultichainDeposit(
         EventEmitter eventEmitter,
         address token,
-        address virtualAccount,
+        address account,
         uint256 amount,
-        uint256 sourceChainId
+        uint256 multichainId
     ) internal {
         EventUtils.EventLogData memory eventData;
 
         eventData.addressItems.initItems(2);
         eventData.addressItems.setItem(0, "token", token);
-        eventData.addressItems.setItem(1, "virtualAccount", virtualAccount);
+        eventData.addressItems.setItem(1, "account", account);
 
         eventData.uintItems.initItems(2);
         eventData.uintItems.setItem(0, "amount", amount);
```

Comment:
wondering it if makes sense to only pass deposit key to this event instead of all other fields

events consume quite a lot of gas, unless i'm missing something account/token/amount could be retrieved from corresponding DepositCreated event later

but tbh i'm not sure if the difference worth it